apps/centraldashboard/upstream/base/deployment.yaml

```diff
@@ -41,7 +41,7 @@ spec:
         - name: DASHBOARD_CONFIGMAP
           value: CD_CONFIGMAP_NAME_PLACEHOLDER
         - name: LOGOUT_URL
-          value: '/authservice/logout'
```

Comment:
This File is copied from upstream and must be changed in the kubeflow/kubeflow repository. We shall not modify upstream folders manually.